code/modules/reagents/chemistry_reagents/medical.dm

```diff
@@ -347,6 +347,22 @@
 		C.disabilities = 0
 	return ..()
 
+/datum/reagant/medicine/roulettium
+	name = "Roulettium"
+	description = "The pure, concentrated essence of unga. Unsafe to ingest in any quantity."
+	color = "#C8A5DC" // rgb: 200, 165, 220
+	taste_description = "poor life choices"
+	custom_metabolism = REAGENTS_METABOLISM * 0.25 //0.1u
+
+/datum/reagent/medicine/roulettium/on_mob_life(mob/living/L, metabolism)
+	L.reagent_shock_modifier += PAIN_REDUCTION_FULL
+	L.adjustToxLoss(-20 * REM)
+	L.heal_limb_damage(0, 20 * REM)
+	L.heal_limb_damage(20*REM, 0)
```

Comment:
Combine these

Reply:
Will do once I figure out what the hell the invisible error is affecting that specific reagent,